roles/kubernetes-apps/csi_driver/upcloud/templates/upcloud-csi-setup.yml.j2

```diff
@@ -183,3 +183,14 @@ roleRef:
   kind: ClusterRole
   name: csi-upcloud-resizer-role
   apiGroup: rbac.authorization.k8s.io
+
+ ---
+ 
+  {% if upcloud_csi_enable_volume_snapshot %}
+  apiVersion: snapshot.storage.k8s.io/v1
+  kind: VolumeSnapshotClass
+  metadata:
+    name: upcloud-csi-snapshotclass
+  driver: storage.csi.upcloud.com
+  deletionPolicy: Delete
+  {% endif %}
```

Comment:
The indentation looks off here compare to the rest of the file and the yaml doc separator should also be conditional.